src/scenario.cpp

```diff
@@ -1857,18 +1860,17 @@ int createStringTable(char * inputString, char *** stringList, int *sizeOfList)
     *stringList = NULL;
     *sizeOfList = 0;
 
+    /* FIXME: temporary workaround: needs rewrite */
+    char *p = const_cast<char*>(strchr(inputString, ','));
```

Comment:
I have a rewrite of this method (porting it to std::vector) sitting around, I'll try to get it back in. In the meanwhile, this is at least safe - if very hacky.